### ISSUE TITLE
Filtering on an event with parameters doesn't allow viewing other events in the segment

### ISSUE THREAD

**Issue by NathanC** [plausible/analytics#2396]
### Past Issues Searched

- [X] I have searched open and closed issues to make sure that the bug has not yet been reported

### Issue is a Bug Report

- [X] This is a bug report and not a feature request, nor asking for self-hosted support

### Using official Plausible Cloud hosting or self-hosting?

Plausible Cloud from plausible.io

### Describe the bug

This is either a bug or I'm misunderstanding the feature, and if so, I apologize.

When I filter on the "sign up" on the Plausible.io demo page (https://plausible.io/plausible.io?goal=Signup), it only shows that single goal in the goal section. It doesn't show other goals that were met in the same session/context as this goal being met.

For example, I couldn't see how many people who signed up also signed up for the newsletter, or also (just an example) scrolled all the way to the bottom of the landing page.

### Expected behavior

Expected behavior would be showing what % of

This seems to happen for filtering on an event that doesn't have parameters, such as newsletter sign up on the demo page: https://plausible.io/plausible.io?goal=Newsletter+signup

Note that goal conversions on the bottom show 2 goals, visiting the blog and sign up, as opposed to the 1 (with breakdown) goal I linked above when filtering on the "sign up" event.

tl;dr; when an event includes parameters, and thus a paramater breakdown UX, it doesn't seem to show other events/goals in that segment.

### Environment

```markdown
- OS: Android 
- Browser: Firefox
- Browser Version: 106.1.0
```

**Comment by NathanC** [comment]
@metmarkosaric Thanks for your response! Those links are very interesting, I'll dig deeper into the discussions later-- it's nice to know this is on the radar. Custom events as they exist currently are pretty flexible, so I think they should suffice for most of what I need.

**Comment by NathanC** [comment]
@metmarkosaric Just curious-- currently, what is the main barrier to seeing other custom events when one custom event is selected? I'm guessing that custom events aren't attached to any sort of session-over-time, but instead they're isolated data points? If so, is that mainly due to technical constraints or intentional for privacy?

(no need for a quick response, just curious to understand the underlying data model better)

**Comment by ukutaht** [comment]
Sorry I don't fully understand your report. When you filter by goal: Signup, we show only that goal since it's the one that's been filtered. What else would you like to see?

There's always a weird contention in Plausible where some filters act on events (i.e. pageviews, custom events) while others apply for the whole session. There's nothing in the data model stopping the goal from being a session-level filter but it could confuse some other reports... Are you expecting it to apply for the whole session, and show other goals within that session?
